### PR TITLE
Use tango-libs docker [1h]

### PR DESCRIPTION
**Pull request by Ingvord**
Refactor Dockerfile to use [tango-libs](https://github.com/tango-controls/tango-libs-docker) docker as base image

Add travis-CI

### CODE REVIEW

File: supervisord.conf [hunk 1]
```diff
@@ -0,0 +1,23 @@
...
+  exec -- /usr/local/bin/Starter $HOSTNAME'
+autorestart=true
+user=tango
+priority=1
+
+[program:tango-test]
+command=/bin/bash -c '
+  /usr/local/bin/tango_admin --add-property sys/tg_test/1 VIP Whoa! &&
+  exec -- /usr/local/bin/TangoTest test'
+autorestart=true
```
Comment (bourtemb) [inline supervisord.conf]: Is it a good idea to autorestart TangoTest?
What if you want to test what happens when the device server is stopped with device admin kill command?
If I understand well, supervisord will restart TangoTest/test automatically in this situation, which can be quite confusing/surprising for the user who explicitly requested the device server to be killed.
`autorestart=unexpected` might be a better choice? What do you think?

Comment (Ingvord) [reply supervisord.conf]: @bourtemb , good point! I did not know about `autorestart=unexpected`, worth giving it a try.

File: .travis.yml [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+
+before_deploy:
+  - docker login -u "$REGISTRY_USER" -p "$REGISTRY_PASS"
+  - docker tag ${DOCKER_IMAGE} ${DOCKER_IMAGE}:${TANGO_VER}
+
+deploy:
+  - provider: script
+    script: docker push ${DOCKER_IMAGE}:latest
+    on:
+      all_branches: true
```
Comment (bourtemb) [inline .travis.yml]: I would deploy only from the master branch, (and maybe on tags?).
I think you could get some strange side effects if someone works in a branch different than master and the docker push is triggered on a buggy version.

Comment (Ingvord) [reply .travis.yml]: Sure!

**Pull request closed without merging**